pb.py

```diff
@@ -26,7 +26,7 @@ def parseTemplateInformation(line):
         key = key.lstrip()
         value = paramArr[1].rstrip()
         value = value.lstrip()
-        print("Param: (key) "+key+", (value) "+value) #debug
+        #print("Param: (key) "+key+", (value) "+value) #debug
```

Comment:
I'm going to leave it for now, as I just pull in the latest release. Don't really see the need as I will be pushing another release version of PyBuilder that finishes the feature that uses that. We just needed one of the other features for building in the meantime otherwise I wouldn't have added it to this PR.